src/test/java/it/pagopa/transactions/services/TransactionServiceTests.java

```diff
@@ -345,7 +351,21 @@ void shouldReturnTransactionInfoForSuccessfulAuthAndClosure() {
 
         Mockito.when(closureSendProjectionHandler.handle(any()))
                 .thenReturn(Mono.just(closedTransactionDocument));
-
+        Mockito.when(
+                eventStoreRepositoryAuthCompletedData.findByTransactionIdAndEventCode(
+                        transactionId.value().toString(),
+                        TransactionEventCode.TRANSACTION_AUTHORIZATION_COMPLETED_EVENT
+                )
+        )
+                .thenReturn(Mono.empty());
```

Comment:
This method tests the successful auth case -> should be something instead of an empty mono?

Reply:
This is the test for the first PATCH call, in this case no `TRANSACTION_AUTHORIZATION_COMPLETED_EVENT` is found into eventstore.